src/mobile/apps/fair/test/client/exhibitors.test.js

```diff
@@ -12,7 +12,7 @@ const sinon = require("sinon")
 const benv = require("benv")
 const { resolve } = require("path")
 
-describe("Exhibitors page client-side code", function () {
+xdescribe("Exhibitors page client-side code", function () {
```

Comment:
intentional?

Reply:
Yeah this kept failing (is failing in master right now too). These specs are dead code anyway.

Reply:
fixes for flakey tests here: https://github.com/artsy/force/pull/6440

Reply:
@eessex was this introduced through decaffination?